tests/test_scoring/test_confidence.py

```
@@ -114,8 +114,11 @@ def test_mmlu_style_grouping(self):
         Within each subject, scores are correlated (all high or all low)."""
         scores = []
         clusters = []
-        for subj, acc in [("math", 0.9), ("history", 0.6), ("science", 0.8), ("art", 0.5)]:
-            rng = np.random.default_rng(hash(subj) % 2**32)
+        for (subj, acc), seed in zip(
+            [("math", 0.9), ("history", 0.6), ("science", 0.8), ("art", 0.5)],
+            [17, 31, 53, 97],
+        ):
```

Ruff (0.15.14)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)